package.json

```diff
@@ -66,6 +66,7 @@
     "diff": "~3.0.0",
     "ipaddr.js": "~1.2.0",
     "isomorphic-fetch": "^2.2.1",
+    "jed": "^1.1.1",
```

Comment:
@amirfefer would mind trying to create a packaging pr for jed? /cc @ekohl if you have any guidance ?

Reply:
@amirfefer bump on this point?

Reply:
seems to be resolved at https://github.com/theforeman/foreman-packaging/pull/2985